hack/test-cmd.sh

```diff
@@ -276,4 +268,93 @@ echo
 wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/metrics" "metrics: " 0.25 80
 echo
 echo
+echo "openshift: ok"
+
+# Atomic Enterprise tests *****************************************************
+# create master config as atomic-enterprise just to test it works
+kill -TERM $OS_PID
+rm -rf ${SERVER_CONFIG_DIR} ${HOME}/.kube
+unset KUBECONFIG
+
+atomic-enterprise admin ca create-master-certs \
```

Comment:
testing the executable name -> enabled/disabled feature set can be done separately in a unit test